OneDrive-Cloud-Player/App.xaml

```diff
@@ -2,11 +2,9 @@
     x:Class="OneDrive_Cloud_Player.App"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:OneDrive_Cloud_Player"
-    xmlns:vm="using:OneDrive_Cloud_Player.ViewModels"
     RequestedTheme="Dark"
     >
     <Application.Resources>
-        <vm:ViewModelLocator xmlns:vm="using:OneDrive_Cloud_Player.ViewModels" x:Key="Locator" />
+
```

Comment:
can we remove <Application.Resources> now also?

Reply:
yes